source/strings.tex

```diff
@@ -824,6 +824,7 @@
       constexpr explicit basic_string(const T& t, const Allocator& a = Allocator());
     constexpr basic_string(const charT* s, size_type n, const Allocator& a = Allocator());
     constexpr basic_string(const charT* s, const Allocator& a = Allocator());
+    constexpr basic_string(nullptr_t) = delete;
```

Comment:
I should have asked this during the review, but why are these `constexpr`?

Reply:
"constexpr all the things" is a life style! :-)

Reply:
I thought this was odd until the contributor who prepared our PR for this decided to strengthen `noexcept` as well and raised my bar for weird.

Reply:
I think neither `constexpr` nor `noexcept` make sense / changes anything for deleted functions.

Reply:
I've created #4706 for this.

Reply:
I thought it made for a nice alignment.

Reply:
@jwakely there you have it ^^^ :-)